src/bms-state.lisp

```diff
@@ -0,0 +1,70 @@
+(defun bms-state-loop () {
+    (var bms-vmin-limit-start (conf-get 'bms-vmin-limit-start))
+    (var bms-vmax-limit-start (conf-get 'bms-vmax-limit-start))
+    (var bms-tmax-limit-start (- (conf-get 'bms-t-limit-start) 3)) ; start 3 degrees before we start limiting
+    (var bms-cell-balance-start 0.5)
```

Comment:
0.5V is too much for sure. You had 0.05V in the previous version. Did we get this number confirmed by some battery people?